bqemulatormanager/manager.py

```diff
@@ -1,4 +1,4 @@
-from typing import Dict, List
+from typing import Dict, List, Union
```

Comment:
[imo] 

I like python 3.10 new syntax `Foo | None` rather than`Union[Foo, None]`.

And we can remove all of these type imports and just use `dict` `list` since python 3.9

This library suppors python 3.8, but these new features are available on python 3.8 with `from __future__ import annotations`

Reply:
but could be another PR